projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/ProjectorStatusWidget.kt

```diff
@@ -125,12 +143,25 @@ class ProjectorStatusWidget(project: Project)
     action.actionPerformed(event)
   }
 
+  @Suppress("unused")
+  private fun clientObserver(newValue: Int) {
+    clients = newValue
+    SwingUtilities.invokeLater { update() }
+  }
+
   companion object {
-    val ID : String by lazy { ProjectorStatusWidget::class.java.name }
+    val ID: String by lazy { ProjectorStatusWidget::class.java.name }
     private fun getIcon(path: String): Icon = IconLoader.getIcon(path, ProjectorStatusWidget::class.java)
     private val ACTIVATION_NEEDED_SIGN: Icon by lazy { getIcon("/META-INF/activationNeeded.svg") }
     private val RUNNING_SIGN: Icon by lazy { getIcon("/META-INF/runningSign.svg") }
     private val STARTING_SIGN: Icon by lazy { getIcon("/META-INF/startingSign.svg") }
     private val DISABLED_SIGN: Icon by lazy { getIcon("/META-INF/disabledSign.svg") }
   }
+
+  override fun update(p0: Observable?, p1: Any?) {
```

Comment:
Let's use more meaningful argument names, at least `o` and `arg` like in [javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/Observer.html#update-java.util.Observable-java.lang.Object-)